// webpack/webpack.config.js
const rules = [
    module.exports = {
        module: {
          rules: [
            {
              test: /\.s(c|a)ss$/,
              use: [
                'vue-style-loader',
                'css-loader',
                {
                  loader: 'sass-loader',
                  // Requires sass-loader@^7.0.0
                  options: {
                    implementation: require('sass'),
                    indentedSyntax: true // optional
                  },
                  // Requires >= sass-loader@^8.0.0
                  options: {
                    implementation: require('sass'),
                    sassOptions: {
                      indentedSyntax: true // optional
                    },
                  },
                },
              ],
            },
          ],
        }
      }
]

const GoogleFontsPlugin = require("google-fonts-webpack-plugin")
 
module.exports = {
    "entry": "index.js",
    /* ... */
    plugins: [
        new GoogleFontsPlugin({
            fonts: [
                { family: "Source Sans Pro" },
                { family: "Roboto", variants: [ "400", "700italic" ] }
            ]
            /* ...options */
        })
    ]
}
